Revit_Engine/Query/UniqueIds.cs

```diff
@@ -35,6 +35,7 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [Deprecated("3.1", "BH.Engine.Adapters.Revit.Query.UniqueIds is an unused duplicate of BH.Engine.Adapters.Revit.Query.UniqueId.")]
         [Description("Returns Revit UniqueIds of given BHoMObjects (stored in CustomData).")]
         [Input("bHoMObjects", "Collection of BHoMObjects")]
```

Comment:
This input attribute is the one flagged by the compliance report

Reply:
I have just fixed it, but I am not sure if I like the fact of being forced to change the code that is neither originated nor changed by me at any point.

Reply:
At the time of PR all files should be compliant, so we run the checks on changed files and, occasionally, this may mean you have to fix legacy incompliance. Once your toolkit is mostly compliant however, this will become less of a problem :smile: